reporter/html.js

```diff
@@ -779,18 +779,22 @@ export function escapeText( s ) {
 			if ( actual !== expected ) {
 
 				message += "<tr class='test-actual'><th>Result: </th><td><pre>" +
-				escapeText( actual ) + "</pre></td></tr>";
-
-			// Don't show diff if actual or expected are booleans
-				if ( !( /^(true|false)$/.test( actual ) ) &&
-					!( /^(true|false)$/.test( expected ) ) ) {
+					escapeText( actual ) + "</pre></td></tr>";
+
+				if ( "number" === typeof details.actual && "number" === typeof details.expected ) {
```

Comment:
but if `details.actual` is `42` and `details.expected` is `NaN`, `diff` will become NaN only and won't really tell what is going on, right?